src/main/proto/envoy/type/v3/token_bucket.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package envoy.type.v3;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
```

Buf (1.54.0)

import "udpa/annotations/status.proto": file does not exist

(COMPILE)